### ISSUE TITLE
Docs are out of date for v4

### ISSUE THREAD

**Issue by gsteel** [laminas/laminas-stratigility#60]
For some reason the v4 docs aren't getting published. Pointers appreciated

**Comment by gsteel** [comment]
CC @froschdesign

**Comment by froschdesign** [comment]
For some reason, the pages for version 4 are not being integrated or recognised by MkDocs:

```bash
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - migration/preparing-for-v3.md
  - v4/api.md
  - v4/creating-middleware.md
  - v4/error-handlers.md
  - v4/executing-middleware.md
  - v4/install.md
  - v4/intro.md
  - v4/middleware.md
  - v4/migration.md
  - v4/usage.md
```

https://github.com/laminas/laminas-stratigility/actions/runs/9981173712/job/28180097748#step:3:42

**Comment by gsteel** [comment]
Yes… I saw that, but the pages _are_ in the nav configuration: https://github.com/laminas/laminas-stratigility/blob/4.1.x/mkdocs.yml

**Comment by froschdesign** [comment]
I can create the documentation locally and everything is there. :man_shrugging:

**Comment by froschdesign** [comment]
Fixed with #61

**Comment by froschdesign** [comment]
It looks like the different indents in `mkdocs.yml` file are causing some problems.
